src/components/NFTPortfolio/Item.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <NuxtLink
+    v-if="getNFTClassMetadataById(classId)"
```

Comment:
just use `NFTClassMetadata`?
```suggestion
    v-if="NFTClassMetadata"
```

Reply:
Maybe `NFTClassMetadata.name` ? since `NFTClassMetadata` is object.